nixos/modules/system/boot/unl0kr.nix

```diff
@@ -15,31 +15,40 @@ in
       description = ''Whether to enable the unl0kr on-screen keyboard in initrd to unlock LUKS.'';
     };
 
-    package = lib.mkPackageOption pkgs "unl0kr" { };
+    package = lib.mkPackageOption pkgs "buffybox" { };
 
     allowVendorDrivers = lib.mkEnableOption "load optional drivers" // {
       description = ''Whether to load additional drivers for certain vendors (I.E: Wacom, Intel, etc.)'';
     };
 
+    useUpstreamPasswordAgent = lib.mkEnableOption "use unl0kr's own password agent" // {
+      description = ''By enabling this, unl0kr will instead be invoked by a bespoke agent written in C instead of a shell script.'';
+    };
+
```

Comment:
What? Absolutely not. Just use the upstream one. The scripted one here is just as bespoke, except it's more maintenance burden on us.

Reply:
Sure

Reply:
done